api/product.rb

```diff
@@ -16,6 +16,7 @@
 require 'api/product/version'
 require 'google/logger'
 require 'compile/core'
+require 'overrides/resources'
```

Comment:
Is this needed?

Reply:
ah, probably not. Might have snuck in with some of my other attempts :P